examples/Cargo.toml

```diff
@@ -15,10 +16,10 @@ bytesize = "1.1.0"
 clap = { version = "4.0.9", features = ["derive"] }
 futures = "0.3.21"
 indicatif = "0.17.0"
-iroh-car = { path = "../iroh-car" }
-iroh-resolver = { path = "../iroh-resolver" }
-iroh-rpc-client = { path = "../iroh-rpc-client" }
-iroh-util = { path = "../iroh-util" }
+iroh-car = { version = "0.1.1", path = "../iroh-car" }
+iroh-resolver = { version = "0.1.1", path = "../iroh-resolver" }
+iroh-rpc-client = { version = "0.1.1", path = "../iroh-rpc-client" }
+iroh-util = { version = "0.1.1", path = "../iroh-util" }
```

Comment:
aren't we doing "0.1.2", or is this an artifact of being the first release?

Reply:
this is just noting the current version, not the new one